### PR TITLE
v1.12 dev -> master

### PR DESCRIPTION
**Pull request by ewels**
Merging `dev` branch to `master` for the v1.12 release.

### CODE REVIEW

**Review by apeltzer — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+## Expected behaviour
+
+<!-- A clear and concise description of what you expected to happen. -->
+
+## System
+
+- Hardware: <!-- [e.g. HPC, Desktop, Cloud...] -->
+- Executor: <!-- [e.g. slurm, local, awsbatch...] -->
+- OS: <!-- [e.g. CentOS Linux, macOS, Linux Mint...] -->
+- Version <!-- [e.g. 7, 10.13.6, 18.3...] -->
```
Comment (Zethson) [inline .github/ISSUE_TEMPLATE/bug_report.md:36]: Maybe we should add the Python version? This is the template for Tools after all and not for the pipelines.

**Review by Zethson — APPROVED** [review]
I would suggest to add a general_question.md issue template, but LGTM

**Review by KevinMenden — APPROVED** [review]

**Review by drpatelh — APPROVED** [review]
All looks great to me!! 😍

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (ewels) [inline .github/ISSUE_TEMPLATE/bug_report.md:36]:
```suggestion
- Version of nf-core/tools: <!-- [e.g. 1.1, 1.5, 1.8.2...] -->
- Python version: <!-- [e.g. 3.7, 3.8...] -->
```

**Pull request merged**